x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/engine/engine_schema.tsx

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useEffect } from 'react';
+
+import { useActions, useValues } from 'kea';
+
+import { EuiBasicTable, EuiBasicTableColumn } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+import { EnterpriseSearchSchemaField } from '../../../../../common/types/engines';
+import { EngineViewTabs } from '../../routes';
+import { EnterpriseSearchEnginesPageTemplate } from '../layout/engines_page_template';
+
+import { EngineIndicesLogic } from './engine_indices_logic';
+
+import { EngineViewLogic } from './engine_view_logic';
+
+export const EngineSchema: React.FC = () => {
+  const { engineName } = useValues(EngineIndicesLogic);
+  const { isLoadingEngineSchema, schemaFields } = useValues(EngineViewLogic);
+  const { fetchEngineSchema } = useActions(EngineViewLogic);
+  const columns: Array<EuiBasicTableColumn<EnterpriseSearchSchemaField>> = [
+    {
+      field: 'field_name',
+      name: i18n.translate('xpack.enterpriseSearch.content.engine.schema.field_name.columnTitle', {
+        defaultMessage: 'Field name',
+      }),
+    },
+    {
+      field: 'field_type',
+      name: i18n.translate('xpack.enterpriseSearch.content.engine.schema.field_type.columnTitle', {
+        defaultMessage: 'Field Type',
+      }),
+    },
+  ];
+
+  useEffect(() => {
+    fetchEngineSchema({ engineName });
+  }, [engineName]);
+  return (
+    <EnterpriseSearchEnginesPageTemplate
+      pageChrome={[engineName]}
+      pageViewTelemetry={EngineViewTabs.SCHEMA}
+      isLoading={isLoadingEngineSchema}
+      pageHeader={{
```

Comment:
we'll want to add `<EngineViewHeaderActions />` to the `rightSideItems` of the `pageHeader` as well.

Reply:
Question: Do we need to add Delete [engine](https://github.com/elastic/kibana/blob/main/x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/engine/engine_view_header_actions.tsx)  feature in Schema table page as well ?

As per the [design](https://www.figma.com/file/IupYoBQ7qMh0dMBGGz2kGq/Introducing-Engines?node-id=590%3A185029&t=GukiyvxR0h56FtRF-0) it doesn't have that. I might be wrong here though.

Reply:
Thats a good question, I just assumed it would be on every engine page, but lets double check with @julianrosado

Reply:
Thanks for the call out @TattdCodeMonkey @saarikabhasi ! 

**TL;DR** - No utility menu on any of the subpages now, only Overview.

**But why tho?** - With the removal of the tabs, we had to title each page as their engine subpage (vs having big page title -> engine name, small tab title -> subpage). With that, the context of that top menu has changed from relating to the capital E Engine to relating to the lowercase s subpage. 

That utility menu was removed from the subpages as a result, since deleting the engine from say a schema page would no longer make sense. It will only be on the Overview page, as that most directly relates to the engine as a whole.

Reply:
I'll push changes without delete actions

Reply:
Let's remove it from indices in another PR, and we can add it to the Overview page in that PR since it's not there currently.

Reply:
It **is** on the overview page already. totally missed that somehow, not sure what I was looking at yesterday. Still think we can remove from the indices page in a small PR on it's own for conciseness.